extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureStorage.java

```diff
@@ -62,12 +62,15 @@ public class AzureStorage
   private static final Logger log = new Logger(AzureStorage.class);
 
   private final AzureClientFactory azureClientFactory;
+  private String defaultStorageAccount;
 
   public AzureStorage(
-      AzureClientFactory azureClientFactory
+      AzureClientFactory azureClientFactory,
+      String defaultStorageAccount
```

Comment:
Do we need this default storageAccount, or can we just always be explicit about what storageAccount we are using on each  method call that this class provides?